### ISSUE TITLE
Need a test case with a comment

### ISSUE THREAD

**Issue by khatchad** [ponder-lab/Null-to-Optional-Eclipse-Plugin#91]
In:
```java
class C {
    Object o = null; // this is a comment
}
```
Out:
```java
import java.util.Optional;
class C {
    Optional<Object> o = Optional.empty(); // this is a comment
}
```

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/274838528047159/820633371417607)

**Comment by orenwf** [comment]
@walleXD We need a test case with a block comment.

**Comment by walleXD** [comment]
Complete fix in #100

**Comment by walleXD** [comment]
Tests and merged with #100
